src/components/Button/Button.scss

```
@@ -392,6 +392,11 @@ $stacking-order: (
       opacity: 0.4;
     }
 
+    // Prevents the loading spinner from overlapping the button’s text, while retaining its width.
+    &.loading .Text {
+      opacity: 0;
```

^This is a [known issue](https://github.com/Shopify/polaris-react/issues/1212) 💯 

```suggestion
      visibility: hidden;
```

---

Ah, I thought we wouldn’t actually want to hide the text from screen readers but seems like Devon recommends to hide it 👍